service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ComplyWithDirectionsControllerTest.java

```diff
@@ -70,37 +71,76 @@ void aboutToStartCallbackShouldAddAllPartiesDirectionsIntoSeparateRoleCollection
+    void aboutToSubmitShouldAddResponseToStandardDirectionOrderWhenEmptyServedCaseManagementOrders() throws Exception {
         UUID uuid = randomUUID();
+        List<Element<Direction>> directions = directions(uuid);
+        Order sdo = order(uuid);
 
-        List<Element<Direction>> directions = ImmutableList.of(Element.<Direction>builder()
-            .id(uuid)
-            .value(Direction.builder()
-                .response(DirectionResponse.builder()
-                    .complied("Yes")
+        CallbackRequest request = CallbackRequest.builder()
+            .caseDetails(CaseDetails.builder()
+                .data(createCaseDataMap(directions)
+                    .put("standardDirectionOrder", sdo)
                     .build())
                 .build())
-            .build());
-
-        Order sdo = Order.builder().directions(ImmutableList.of(Element.<Direction>builder()
-            .id(uuid)
-            .value(Direction.builder()
-                .directionType("example direction")
-                .build())
-            .build()))
             .build();
 
+        CaseData caseData = makeRequest(request, "about-to-submit");
+
+        assertThat(caseData.getStandardDirectionOrder().getDirections().get(0).getValue().getResponses()).isNotEmpty();
+    }
+
+    @SuppressWarnings("unchecked")
+    @Test
+    void aboutToSubmitShouldAddResponseToCaseManagementOrderWhenPopulatedServedCaseManagementOrders() throws Exception {
+        UUID uuid = randomUUID();
+        List<Element<Direction>> directions = directions(uuid);
+        Order sdo = order(uuid);
+
         CallbackRequest request = CallbackRequest.builder()
             .caseDetails(CaseDetails.builder()
                 .data(createCaseDataMap(directions)
                     .put("standardDirectionOrder", sdo)
+                    .put("servedCaseManagementOrders", caseManagementOrders(uuid))
                     .build())
                 .build())
             .build();
 
         CaseData caseData = makeRequest(request, "about-to-submit");
 
-        assertThat(caseData.getStandardDirectionOrder().getDirections().get(0).getValue().getResponses()).isNotEmpty();
+        assertThat(getResponses(caseData.getServedCaseManagementOrders().get(0).getValue())).isNotEmpty();
+    }
+
+    private List<Element<DirectionResponse>> getResponses(CaseManagementOrder order) {
+        return order.getDirections().get(0).getValue().getResponses();
+    }
+
+    private List<Element<CaseManagementOrder>> caseManagementOrders(UUID uuid) {
+        return ImmutableList.of(Element.<CaseManagementOrder>builder()
+            .value(CaseManagementOrder.builder()
+                .directions(directions(uuid))
+                .build())
+            .build());
+    }
+
+    private Order order(UUID uuid) {
+        return Order.builder().directions(ImmutableList.of(Element.<Direction>builder()
+                .id(uuid)
```

Comment:
Do we need to pass / set the id prop at all? Doesn't look like you check it

Reply:
directions update by UUID